src/transformState.js

```diff
@@ -46,7 +46,29 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const attribute of transforms) {
+    switch (attribute.operation) {
+      case 'addProperties': {
+        for (const [addKey, addValue] of Object.entries(attribute.properties)) {
+          state[addKey] = addValue;
+        }
+        break;
+      }
+
+      case 'removeProperties': {
+        for (const removeKey of attribute.properties) {
```

Comment:
не нужно привязываться к типу операции, просто key
```suggestion
        for (const key of attribute.properties) {
```